### PR TITLE
Designate technical poc 159495040

### PR DESCRIPTION
**Pull request by richard-dds**
Allow a MO to designate themselves as a technical POC. This involves a kind of gross addition to the (already gross) JEDIRequestFlow class that modules the request body data before updating. My opinion is that this is ok for now, but once we finalize the Request body schema we should tackle the big hunk of tech debt known as `atst.routes.requetsts`.

screen shot 2018-08-14 at 4 32 48 pm

screen shot 2018-08-14 at 4 32 55 pm

### CODE REVIEW

**Comment by luisgov** [comment]
@richard-dds maybe this should be a checkbox instead?

**Comment by richard-dds** [comment]
@luisgov Made the input into a checkbox.

File: js/components/checkbox_input.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default {
+  name: 'checkboxinput',
```
Comment (dandds) [inline js/components/checkbox_input.js]: 🎊

File: tests/routes/test_request_new.py [hunk 2]
```diff
@@ -103,6 +101,58 @@ def test_non_creator_info_is_not_autopopulated(monkeypatch, client, user_session
...
+
+
+def test_not_am_poc_allows_user_to_fill_in_poc_info(client, user_session):
+    creator = UserFactory.create()
+    user_session(creator)
+    request = RequestFactory.create(creator=creator, body={})
+    response = client.post(
+        "/requests/new/3/{}".format(request.id),
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+        data="am_poc=no&fname_poc=test&lname_poc=user&email_poc=test.user@mail.com&dodid_poc=1234567890",
```
Comment (dandds) [inline tests/routes/test_request_new.py]: I might use `urllib.parse.urlencode` on a dictionary instead to make it clearer to read/update.

File: tests/routes/test_request_new.py [hunk 3]
```diff
@@ -103,6 +101,58 @@ def test_non_creator_info_is_not_autopopulated(monkeypatch, client, user_session
...
+    )
+    assert ERROR_CLASS not in response.data.decode()
+
+
+def test_not_am_poc_allows_user_to_fill_in_poc_info_for_new_request(client, user_session):
```
Comment (dandds) [inline tests/routes/test_request_new.py]: Is this test name right?

**Pull request merged**